### DISCUSSION TITLE
Replacement display for Newt?

### DISCUSSION THREAD

**Discussion by kava60** [Phambili-Tech/Newt_Display#58] [Q&A] [answered]
Somehow, something hit the display on my Newt - it must have been something pointy like a small screwdriver or something and the tiny impact spot caused an area of permanently black pixels horizontally from the impact location. It still works but of course the dead area is quite annoying!

![SadNewt](https://user-images.githubusercontent.com/67206957/220807236-cae4c2c2-ace7-405c-a028-e824baeb84f1.jpg)

Can someone provide the part number to reorder a replacement (and a source that carries it)? Also how is the display adhered to the PCB? I gently pulled but it seems to be affixed quite securely... No ifixit for the Newt yet to tell me how :-)

Regards,
  Kurt

**Comment by darianbjohnson** [comment, marked as answer]
Kurt - The Newt uses Sharp Display LS027B7DH01. You can find them at Digikey here - https://www.digikey.com/en/products/detail/sharp-microelectronics/LS027B7DH01/5054066

The older version - LS027B7DH01A - should work as well (and it will save you $5) - though this version has a slightly higher power consumption rate, I think. -https://www.digikey.com/en/products/detail/sharp-microelectronics/LS027B7DH01A/5054067

**Comment by darianbjohnson** [comment]
And we use double-sided tape to attach the screen. You should be able to get a fingernail underneath the screen at lift it up. Note - the flex cable is really delicate....

**Reply by kava60** [reply]
Thank you! Exactly the info I need.

**Comment by chaseadam** [comment]
@kava60 I have a handful of extra screens if you are interested.

**Reply by kava60** [reply]
Thanks for the generous offer but I already have ordered a replacement from digi-key.
